packages/generator/templates/app/src/pages/auth/reset-password.tsx

```diff
@@ -36,7 +35,7 @@ const ResetPasswordPage: BlitzPage = () => {
           }}
           onSubmit={async (values) => {
             try {
-              await resetPasswordMutation({ ...values, token })
+              token && await resetPasswordMutation({ ...values, token })
```

Comment:
thanks for the PR! 

instead of this, we should add an `assert` statement just above this line :)